modules/mcagar/src/main/ts/ephox/mcagar/api/TinyApis.ts

```diff
@@ -158,19 +159,18 @@ export const TinyApis = function (editor: Editor): TinyApis {
     editor.focus();
   });
 
+  const sHasFocus = <T> (expected: boolean) => Step.sync<T>(() => {
+    Assertions.assertEq(() => 'Assert whether editor hasFocus', expected, editor.hasFocus());
+  });
+
   const sNodeChanged = <T> () => Step.sync<T>(function () {
     editor.nodeChanged();
   });
 
-  const sTryAssertFocus = <T> () => Waiter.sTryUntil<T, T>(
-    'Waiting for focus on tinymce editor',
-    FocusTools.sIsOnSelector(
-      'iframe focus',
-      SugarElement.fromDom(document),
-      'iframe'
-    ),
-    100,
-    1000
+  const sTryAssertFocus = <T> (waitTime?: number) => Waiter.sTryUntil<T, T>(
+    'Waiting for focus',
+    sHasFocus(true),
+    waitTime,
```

Comment:
This isn't how the `Waiter` API works. The 3rd argument is the check interval, the 4th argument is the time to wait. So maybe change this to something like this, as we shouldn't need to configure the interval?
```suggestion
  const sTryAssertFocus = <T> (waitTime?: number) => Waiter.sTryUntil<T, T>(
    'Waiting for focus',
    sHasFocus(true),
    50,
    waitTime
```